File: src/AMX101.Site/wwwroot/app/models/populatedClaim.ts
import { ClaimIndustry, Type } from "./enums";

export class PopulatedClaim {
    public id: number;
    public claimName: string;
    public claimValue: string;
    public formattedClaimText: string;
    public industry: ClaimIndustry;
    public postcode: string;
    public threshold: number;
    public sourceId: number;
    public type: Type;
}

File: src/AMX101.Site/Models/PopulatedClaim.cs
using AMX101.Dto.Models;

namespace AMX101.Site.Models
{
    public class PopulatedClaim
    {
        public int Id { get; set; }
        public int? SourceId { get; set; }
        public string ClaimName { get; set; }
        public string ClaimValue { get; set; }
        public string FormattedClaimText { get; set; }
        public Industry Industry { get; set; }
        public string Postcode { get; set; }
        public int Threshold { get; set; }
        public Type Type { get; set; }
        public string ImagePath { get; set; }
    }
}
